### PR TITLE
Add the 'files view' command

### PR DESCRIPTION
**Pull request by KevinLoesch1**

### CODE REVIEW

**Review by gejohnston — COMMENTED** [review]
I approve the changes that I did not implement.

**Review by t1m0thyj — COMMENTED** [review]
LGTM overall, left a few questions around tests 🙂

File: packages/cli/__tests__/zosfiles/__integration__/view/uss/cli.files.view.uss.integration.test.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+*/
+
+import * as path from "path";
+import { ITestEnvironment, runCliScript } from "@zowe/cli-test-utils";
+import { TestEnvironment } from "../../../../../../../__tests__/__src__/environment/TestEnvironment";
+import { ITestPropertiesSchema } from "../../../../../../../__tests__/__src__/properties/ITestPropertiesSchema";
+
+let TEST_ENVIRONMENT: ITestEnvironment<ITestPropertiesSchema>;
+
+describe("List USS file", () => {
```
[the remark this answers is not in the document]
Comment (KevinLoesch1) [reply packages/cli/__tests__/zosfiles/__integration__/view/uss/cli.files.view.uss.integration.test.ts:19]: I'll correct this. I found a couple of other places that had the incorrect description as well. They've been updated with the latest commit.

File: packages/cli/__tests__/zosfiles/__system__/view/ds/View.ds.system.test.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+let defaultSystem: ITestPropertiesSchema;
+let dsname: string;
+const TIMEOUT = 20000;
+
+
+describe("View Data Set", () => {
+    beforeAll(async () => {
+        testEnvironment = await TestEnvironment.setUp({
+            installPlugin: true,
+            testName: "zos_extended_view_ds",
```
[the remark this answers is not in the document]
Comment (KevinLoesch1) [reply packages/cli/__tests__/zosfiles/__system__/view/ds/View.ds.system.test.ts:31]: I've updated the uss and ds versions to match the other commands.

File: packages/cli/__tests__/zosfiles/__system__/view/ds/View.ds.system.test.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
+/*
```
Comment (zFernand0) [inline packages/cli/__tests__/zosfiles/__system__/view/ds/View.ds.system.test.ts:1]: Curious why GH doesn't show the  corresponding `.snap` file (in a PR view) for these tests 🤔 

image

**Review by zFernand0 — APPROVED** [review]
LGTM!

**Pull request merged**